### DISCUSSION TITLE
Request for custom input & output folders

### DISCUSSION THREAD

**Discussion by brbbbq** [Comfy-Org/ComfyUI#2354] [Ideas]
I really appreciate the addition of extra_model_paths.yaml that allows us to define custom locations for our model files.

Currently I use a symbolic link in Windows to point to a custom location for my output folder, but this causes issues when trying to update ComfyUI.

It would be great if the ability to add a custom location for the input & output folder were added to extra_model_paths.yaml.

**Comment by pkpkTech** [comment]
Output folder can be specified by command line arguments.
For example, to make it the outputs folder on the D drive, use the following:
`python main.py --output-directory "D:\outputs"`

**Reply by brbbbq** [reply]
Nice, in my case I opened run_nvidia_gpu.bat, and added "--output-directory "G:\outputs" to the first line.

Thank you very much!
